pkg/pyramid/file_tracker.go

```diff
@@ -0,0 +1,50 @@
+package pyramid
+
+import (
+	"sync"
+
+	"github.com/treeverse/lakefs/pkg/pyramid/params"
+)
+
+// fileTracker tracks file open requests in TierFS to avoid race conditions with cache rejection/eviction
+type fileTracker struct {
+	refMap map[string]int
+	mu     sync.Mutex
+}
+
+type decCallback func(int)
+
+func newFileTracker() *fileTracker {
+	return &fileTracker{
+		refMap: map[string]int{},
+		mu:     sync.Mutex{},
```

Comment:
not needed

Reply:
can remove the mu initialization it is a value type no need initialize it with the same value